augur/metrics/issue/issue.py

```diff
@@ -133,7 +133,7 @@ def issues_first_time_closed(self, repo_group_id, repo_id=None, period='day', be
                     AND action = 'closed'
                     AND repo.repo_id = issues.repo_id
                     AND issues.issue_id = issue_events.issue_id
-                    And issue_events.created_at BETWEEN :begin_date AND :end_date
```

Comment:
have we tested that the issue model still works with this change

Reply:
@sgoggins The issue model would not be affected, this change is just in one of our issue metrics in our api, and the issue model does not hit our own api at all